tests/auto/keras/test_constraints.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+import numpy as np
+from numpy.testing import assert_allclose
+from theano import tensor as T
+
+
+class TestConstraints(unittest.TestCase):
+    def setUp(self):
```

Comment:
I don't know ... I think it's because of habit when using the unittest framework, I think it magically recognised that particular name, so it "has" to be camel case to be discovered appropriately. My knowledge could be stale, but that's why.